src/Controller/EntryController.php

```diff
@@ -627,6 +627,16 @@ private function showEntries($type, Request $request, $page)
         $currentEntryId = $request->attributes->getInt('id');
 
         $formOptions = [];
+        $direction = 'desc';
+        $sortBy = null;
+
+        if (null !== $request->get('entry_filter') && null !== $request->get('entry_filter')['sortType'] && '' !== $request->get('entry_filter')['sortType']) {
```

Comment:
to me display settings should be set only via GET parameters.
so no need for them to be part of the filter form. just well built links with the parameters on the right page will do the job

Reply:
See https://github.com/wallabag/wallabag/pull/8430#discussion_r2288097544